modules/ROOT/pages/to-share-an-asset.adoc

```diff
@@ -0,0 +1,79 @@
+
+== Share With a User
+
+If the person is not in your organization, click *Access Management* > *Users* to add them.
+
+. Click an asset to show its details.
+. Click *Share*.
+. Begin typing the name of the person you want to share this asset with, and select that person in the list.
+. Select whether to grant the user Viewer, Contributor, or Admin rights.
+. Repeat steps 3 and 4 to share with additional users.
+. Click *Save*.
+
+You can also click *Share* to change who you are sharing the asset with or to stop sharing with one or more users.
+
+== Share With Your Organization
+
+Exchange administrators and asset administrators can share an asset to everyone in their organization.
+
+. Click an asset to show its details.
+. Click *Share*.
+. Begin typing the name of your organization.
+. Select the list item for everyone at your organization.
+. Select whether to grant Viewer, Contributor, or Admin rights.
+. Click *Save*.
+
+When an asset is shared with an organization, email notification is sent to the organization administrators.
+
+You can also click *Share* to stop sharing the asset with everyone at the organization.
+
+== Share With an External Organization
+
+Exchange administrators and asset administrators can grant viewer access for an asset to everyone in a chosen external organization.
+
+The organization granting viewer access is the granting organization, and the organization that receives access is the receiving organization.
+
+. The receiving organization adds the granting organization to its "External Access" list in Anypoint Access Management, as described in xref:access-management::external-organization-access.adoc[External Organization Access].
++
+Assets can only be shared with another organization if that organization has enabled receiving viewer access to assets from external organizations.
++
+. The granting organization adds the receiving organization to its "External Access" list in Anypoint Access Management, as described in xref:access-management::external-organization-access.adoc[External Organization Access].
+. Login to your Exchange administrator or asset administrator account in the granting organization.
+. Click an asset to show its details.
+. Click *Share*.
+. Begin typing the name of the external organization you want to share this asset with.
+. Select the organization in the list.
+. Repeat steps 6 and 7 to share with additional organizations.
+. Click *Save*.
+
+When an asset is shared with an organization, email notification is sent to the organization administrators.
+
+You can also click *Share* to change who you are sharing the asset with or to stop sharing with one or more organizations.
+
+Only internal users in the same organization as the asset can request access to an API or gain the contributor or administrator roles.
+
+=== Using Assets Shared with External Organizations
```

Comment:
This heading does not make clear who is using the assets: the grantor or the receiver.
Should it be something like "Using Assets Shared by a Granting Organization"?
My first reading was that this referred to a user in the granting org, rather than the receiving org.
Consider replacing "external" with "receiving" in the following text.

Reply:
Should we have a sentence or two here that lists types of assets shared, or is that covered elsewhere?

Reply:
Replaced "external" with "receiving" and rewrote for clarity. A few weeks ago the Exchange team realized that the Exchange docs needed a list of types of assets that can be shared, so happily that is now on the front page of the Exchange docs.